server/helpers/enforceVerification.js

```diff
@@ -0,0 +1,10 @@
+import responseMessage from './responseMessage';
+
+export default (request, response, next, user) => {
+  const { verified } = user;
+  if (verified || request.originalUrl.slice(0, 19) === '/api/v1/auth/verify') {
+    request.userData = user;
+    return next();
+  }
+  responseMessage(response, 403, { message: 'forbidden access! you must be verified to access this route' });
```

Comment:
Let this message be something like `please verify your account to perform this action`

Reply:
Alright. I will fix that